src/chart/graph/circularLayoutHelper.ts

```diff
@@ -163,11 +184,55 @@ const _layoutNodesBasedOn: Record<'value' | 'symbolSize', LayoutNode> = {
             const radianHalf = halfRemainRadian + _symbolRadiansHalf[node.dataIndex];
 
             angle += radianHalf;
-            node.setLayout([
+            // auto circular layout for not dragged node
+            !node.getFixed() && node.setLayout([
```

Comment:
For the circularLayout, perhaps using
```
node !== draggingNode
```
Can also do the same?

Reply:
Only automatically set circular layout for the node not dragged yet, so should check node's fixed state, not only the dragging node. 😂